keyboards/sporewoh/banime40/readme.md

```diff
@@ -0,0 +1,27 @@
+# banime40 
+
+<img src="https://i.imgur.com/sHQyMfE.jpeg" width="500">
```

Comment:
```suggestion
![](https://i.imgur.com/sHQyMfEh.jpeg)
```

Reply:
I've now updated the image to use markdown over img tags